Source/WebKit/WPEPlatform/wpe/headless/WPEDisplayHeadless.cpp

```diff
@@ -69,6 +71,22 @@ static WPEBufferDMABufFormats* wpeDisplayHeadlessGetPreferredDMABufFormats(WPEDi
 }
 #endif
 
+static WPEKeymap* wpeDisplayHeadlessGetKeymap(WPEDisplay* display, GError** error)
+{
+    static std::once_flag onceFlag;
+    std::call_once(onceFlag, [display]() {
+        auto* priv = WPE_DISPLAY_HEADLESS(display)->priv;
+        priv->m_keymap = adoptGRef(wpe_keymap_xkb_new());
```

Comment:
I wouldn't say this is a dummy keymap, it's a fallback. Since XKB is a common dependency, maybe we can use it always as fallback in `wpe_display_get_keymap`, and document that if platform doesn't implement get_keymap, a default one is created for a standard US 105 keyboard. We don't need call_once for this, this is always expectefd to be called from the main thread only.